tools/lab_dispatch/build_install_deb.py

```diff
@@ -16,37 +34,62 @@ def parse_args():
 
 
 def prepare_repo(repo_root, package_path):
-    shutil.move(package_path / "debian", repo_root)
+    shutil.move(str(package_path / "debian"), str(repo_root))
 
 
+@github_group("Installing build depends")
 def install_build_depends(repo_root):
+    environ = copy(os.environ)
+    environ["DEBIAN_FRONTEND"] = "noninteractive"
+
     subprocess.check_call(
-        ["sudo", "apt-get", "-y", "build-dep", "."], cwd=repo_root
+        [
+            "sudo",
+            "-E",
+            "apt-get",
+            "-y",
+            "build-dep",
+            ".",
+        ],
```

Comment:
So we are running the equivalent of:
```
DEBIAN_FRONTEND=noninteractive sudo -E apt-get...
```
Would it work if we instead ran the equivalent of:
```
sudo DEBIAN_FRONTEND=noninteractive apt-get...`
```
This is usually the suggestion on how to set the non-interactive mode (probably because it's a little safer) and it is consistent with how we usually install packages in other scripts.

Reply:
I don't know if providing the envvars to the Popen args is a good practice actually, let me investigate. This is not great but it is ran on a throw away instance, so it shouldn't cause many issues either

Reply:
done